bentoml/cli/aws_ec2.py

```diff
@@ -0,0 +1,308 @@
+# Copyright 2019 Atalaya Tech, Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import click
+
+from bentoml.utils import status_pb_to_error_code_and_message
+from bentoml.utils.lazy_loader import LazyLoader
+from bentoml.cli.utils import Spinner
+from bentoml.utils import get_default_yatai_client
+from bentoml.cli.click_utils import (
+    BentoMLCommandGroup,
+    parse_bento_tag_callback,
+    _echo,
+    CLI_COLOR_SUCCESS,
+)
+from bentoml.cli.deployment import (
+    _print_deployment_info,
+    _print_deployments_info,
+)
+from bentoml.yatai.deployment import ALL_NAMESPACE_TAG
+from bentoml.exceptions import CLIException
+
+yatai_proto = LazyLoader("yatai_proto", globals(), "bentoml.yatai.proto")
+
+
+def get_aws_ec2_sub_command():
+    # pylint: disable=unused-variable
+
+    @click.group(name="ec2", cls=BentoMLCommandGroup, help="commandas for ec2")
+    def aws_ec2():
+        pass
+
+    @aws_ec2.command(help="Deploy BentoServide to ec2")
+    @click.argument("name", type=click.STRING)
+    @click.option(
+        "-b",
+        "--bento",
+        type=click.STRING,
+        required=True,
+        callback=parse_bento_tag_callback,
+    )
+    @click.option(
+        "--region",
+        type=click.STRING,
+        default="ap-south-1",
```

Comment:
It is better practice to move the default values like region, min-capacity to a central location for easy management and configuration.  You can see the example in Azure function deployment: https://github.com/bentoml/BentoML/blob/master/bentoml/yatai/deployment/azure_functions/constants.py

Reply:
fixed